File: 1897 redistribute-characters-to-make-all-strings-equal/solution.cpp
class Solution {
public:
    bool makeEqual(vector<string>& words) {
        vector<int> f(26,0);
        for(string s:words) {
            for(char c:s) f[c-'a']++;
        }
        int val = words.size();
        for(int i:f) {
            if(i%val!=0) return false;
        }
        return true;
    }
};

File: 1897 redistribute-characters-to-make-all-strings-equal/solution.rs
impl Solution {
    pub fn make_equal(words: Vec<String>) -> bool {
        let mut f = vec![0;26];
        for s in &words {
            for c in s.chars() {
                f[(c as u8 - b'a') as usize]+=1;
            }
        }
        let val = words.len();
        for i in f {
            if i%val!=0 {
                return false
            }
        }
        true
    }
}
